CHANGES_UNRELEASED.md

```diff
@@ -1,7 +1,21 @@
 **See [the release process docs](docs/howtos/cut-a-new-release.md) for the steps to take when cutting a new release.**
 
 # Unreleased Changes
- 
+
+ ## General
+
+- The nimbus-sdk repo has been imported (with git history) into
+  `components/nimbus-sdk`.  It is no longer a submodule.  Developers
+  may need to execute
+
+  ```bash
+  rm -fR components/external/nimbus-sdk
+  ```
+
+  This is not expected to have any ramifications for consumers, other than
+  that the `nimbus-sdk` will (in the future) have the same version numbers as
+  the rest of `application-services`.
```

Comment:
Since we were already shipping it via the megazord, I don't think this will actually change from the consumer's perspective - they would already have been consuming it via a package named `org.mozilla.appservices:nimbus` with the same version number as the rest of appservices.

Reply:
OK, I've just shortened it to "This is not expected to have any ramifications for consumers."